tasks/kmt.py

```diff
@@ -1912,3 +1954,77 @@ def _check_status(_):
 
     loop_status(_check_status, timeout_sec=timeout_sec)
     info(f"[+] Setup job {setup_job.name} finished with status {setup_job.status}")
+
+
+# by default the PyYaml dumper does not indent lists correctly using to problem when
+# starting the agent. The following solution is taken from
+# https://stackoverflow.com/questions/25108581/python-yaml-dump-bad-indentation
+class IndentedDumper(yaml.Dumper):
+    def increase_indent(self, flow=False):
+        return super().increase_indent(flow, False)
+
+
+@task
+def install_ddagent(
+    ctx: Context,
+    api_key: str,
+    vms: str | None = None,
+    stack: str | None = None,
+    ssh_key: str | None = None,
+    verbose=True,
+    arch: str | None = None,
+    major: str = "7",
+    minor: str = "55",
```

Comment:
I think it would be better to use `tasks/libs/releasing/version.py:current_version` later in the code to get the current minor version, that way we don't have to change it on every release.

Reply:
Setting version to `last_stable` by default as discussed.